packages/test-utils/src/wrapper.js

```diff
@@ -569,10 +569,19 @@ export default class Wrapper implements BaseWrapper {
       tagName === 'TEXTAREA' ||
       tagName === 'SELECT'
     ) {
-      const event = tagName === 'SELECT' ? 'change' : 'input'
       // $FlowIgnore
       this.element.value = value
-      this.trigger(event)
+
+      if (tagName === 'SELECT') {
+        this.trigger('change')
+      } else {
+        this.trigger('input')
+      }
+
+      // for v-model.lazy, we need to trigger a change event, too.
+      if (tagName === 'INPUT' && this.element._vModifiers) {
```

Comment:
shall we do `vModifiers.lazy` ?

Reply:
Lazy can be used on textarea too :)

Reply:
Oops, I thought that was what I was doing. Yes, we definitely should, I'll fix this.

Reply:
Fixed